shell/platform/darwin/ios/framework/Headers/FlutterViewController.h

```diff
@@ -53,6 +53,12 @@ FLUTTER_EXPORT
 
 - (id<FlutterPluginRegistry>)pluginRegistry;
 
+/**
+ Specifies the launch view to initialize the view with.  If not specified, uses a view
+ generated from `UILaunchStoryboardName` from the main bundle's `Info.plist` file.
+ */
+@property(strong, nonatomic) UIView* launchView;
```

Comment:
How about `splashScreenView`? IMO, the name `launchView` does not make the views purpose immediately clear.

Also, I would amend the docstring to say something like: "Flutters rendering is asynchronous. Due to this, the first frame rendered by the Flutter application may not immediately show up when the UIView is initially placed in the view hierarchy. This view is used a a replacement till the first frame is rendered."

Reply:
I would also add note about how the user should configure the launch/splashScreen view for multiple sizes. Saying that an autoresizing mask is applied should be sufficient.